tests/components/evohome/conftest.py

```diff
@@ -153,3 +158,32 @@ async def setup_evohome(
         assert mock_client.account_info is not None
 
         yield mock_client
+
+
+def get_dhw_entity(hass: HomeAssistant) -> EvoDHW:
+    """Return the DHW entity of the evohome system."""
+
+    broker: EvoBroker = hass.data[DOMAIN]["broker"]
+
+    if (dhw := broker.tcs.hotwater) is None:
+        pytest.fail("DHW expected, but was not found")
+
+    entity_registry = er.async_get(hass)
+    entity_id = entity_registry.async_get_entity_id(
+        Platform.WATER_HEATER, DOMAIN, dhw._id
+    )
+
+    component: EntityComponent = hass.data.get(Platform.WATER_HEATER)  # type: ignore[assignment]
+    return next(e for e in component.entities if e.entity_id == entity_id)  # type: ignore[return-value]
+
+
+@pytest.fixture
+async def dhw(
+    hass: HomeAssistant,
+    config: dict[str, str],
+    install: str,
+) -> AsyncGenerator[EvoDHW]:
+    """Return the first heating zone (Climate entity) of the evohome system."""
```

Comment:
Hmm - I missed this comment first time around. `get_dhw_entity` returns a water heater entity, and not a climate entity?

Reply:
Ah yes, I see now. 

Nice spot!